src/pages/user_profile/UserProfile.jsx

```diff
@@ -27,6 +27,7 @@ export default function UserProfile(props) {
   const [profile, setProfile] = useState({
     profileName: '',
     email: undefined,
+    emailPreference: false,
```

Comment:
Why `false` instead of `undefined`, as is used for `id` and `email`?

I would expect this page's content to block until the profile is loaded, is that not the case here?